docker_push

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#Push built docker image to dockerhub
```

Comment:
It's worth having a `set -e` here so that the script exits if one of the lines fails.

Reply:
Good suggestion - thanks. Have added it